wiki/Beatmaps/Beatmapsets/Guest_difficulty/en.md

```diff
@@ -0,0 +1,10 @@
+---
+tags:
+  - guest difficulties
+---
+
+# Guest difficulty
+
+A **guest difficulty** is a difficulty of a [beatmapset](/wiki/Beatmaps/Beatmapsets) that is not created by the [beatmapset host](/wiki/Beatmaps/Beatmapsets/Beatmapset_host). A guest difficulty featuring multiple mappers is also known as a [collaborative difficulty](/wiki/Beatmaps/Beatmap_collaborations)
```

Comment:
> A guest difficulty featuring multiple mappers is also known as a collaborative difficulty

technically true i guess but the host could also participate in collab which makes this sentence confusing. it sounds like ur talking about strictly multiple guests